tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -28,13 +28,13 @@
 
     <!-- Integration: AWSSDK.Core -->
     <!-- Assembly: AWSSDK.Core -->
-    <!-- Latest package https://www.nuget.org/packages/AWSSDK.Core/3.7.103.4 -->
-    <PackageReference Include="AWSSDK.Core" Version="3.7.103.4" />
```

Comment:
This PR is changing a _lot_ more than just MSTest, we really need to bump all these other versions separately. I have a PR that fixes couchbase here https://github.com/DataDog/dd-trace-dotnet/pull/3596 and OTel here: https://github.com/DataDog/dd-trace-dotnet/pull/3604

Reply:
fixed by the merge commit.